xAPI.md

```diff
@@ -2611,27 +2613,45 @@ can make sense of it.
 ### 4.2 Language Maps
 
 ##### Description
-A language map is a dictionary where the key is a 
-[RFC 5646 Language Tag](http://tools.ietf.org/html/rfc5646), and the value is an 
-string in the language specified in the tag. This map SHOULD be populated as 
-fully as possible based on the knowledge of the string in question in different 
-languages.  
+A language map is a dictionary where the key is a [RFC 5646 Language Tag](http://tools.ietf.org/html/rfc5646), and the value is a string in the language specified in the tag. This map SHOULD be populated as fully as possible based on the knowledge of the string in question in different languages.  There are also places in the specification, 
+such as context, which will require only a language "code" from RFC 5646.
 
 The content of strings within a language map is plain text. It's expected that any formatting code 
 such as HTML tags or markdown will not be rendered, but will be displayed as code when this string is 
 displayed to an end user. An important exception to this is if language map object is used in an extension and 
 the owner of that extension IRI explicitly states that a particular form of code will be rendered.
 
-### 4.2.1 Lang Codes
-
 ### 4.3 IRIs
 
+Internationalized Resource Identifiers, or IRIs, are unique identifiers which could also be resolvable. Because 
+resolving is not a requirement, IRIs/URIs are used instead of IRLs/URLs. In order to allow the greatest flexibility 
+in the characters used in an identifier, IRIs are used instead of URIs as IRIs can contain some characters outside 
+of the ASCII character set. 
+
+IRIs always include a scheme. This is not a requirement of this standard, but part of the definition of IRIs, per RFC 3987. What are sometimes called 'relative IRIs' are not IRIs.
+
+When used as an id, the IRI SHOULD contain a human-readable portion which would allow for disambiguation from 
+other syntactically similar character strings as this portion of the IRI represents semantics.  It is not a token 
+with multiple interpretations.
+
 ### 4.4 UUIDs
 
+Universally Unique Identifiers, or UUIDs, are 128-bit values that are globally unique.  Unlike IRIs, there is 
+no expectation of resolvability as UUIDs take on a completely different format.  UUIDs MUST be in the standard 
+string form.  It is recommended variant 2 in [RFC 4122](http://tools.ietf.org/html/rfc4122) are used.
+
 ### 4.5 ISO 8601 Timestamps
```

Comment:
Very good point - I'm going to make an issue around this because all 3 sections will need slight edits as we disambiguate the timestamp property from the Timestamp data type.